### ISSUE TITLE
[Bug]: Cannot install this plugin with new LTS Node 20 

### ISSUE THREAD

**Issue by rlittolff-lily** [ComfortablyCoding/strapi-plugin-io#60]
### What issue are you experiencing?

When I want to install my strapi with node20, I have this error:

```
error strapi-plugin-io@1.1.4: The engine "node" is incompatible with this module. Expected version ">=12.x.x <=18.x.x". Got "20.8.1"
```

Could you please update or remove node dependencies ?

### Steps To Reproduce

1. Use Node LTS 20.x
2. Try to install strapi-plugin-io
3. You'll get this error

### What version of the plugin are you using?

1.1.4

### What strapi version are you using?

4.15.0

**Comment by ComfortablyCoding** [comment]
Thanks for the report!

Unfortunately I cannot remove it at this time as the `engines` property in the `package.json` is a requirement for Strapi plugins to be accepted to the marketplace.

I will release a patch release over the weekend updating the engines restriction.

**Comment by ComfortablyCoding** [comment]
This should be fixed in v1.1.5
